src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/CreateTask.groovy

```diff
@@ -89,4 +90,18 @@ class CreateTask extends AbstractServerTask {
             return new File(DEFAULT_PATH + fileName)
         }
     }
+
+    void createApplicationFolders() {
+        File serverDir = getServerDir(project)
+        File appsDirectory = new File(serverDir, 'apps')
+        File dropinsDirectory = new File(serverDir, 'dropins')
+
+        if (!appsDirectory.exists()) {
+            appsDirectory.mkdir()
```

Comment:
If there is a problem creating the directory, does this throw an exception? Or is there a return value that can be checked?